packages/agent/src/agents/core/utils.ts

```
@@ -136,6 +227,7 @@ export function truncateToolResults(
         typeof msg.content === 'string'
       ) {
         msg.content = truncateStringContentHelper(msg.content, maxLength);
+        msg;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)